final/design/commands.tex

```diff
@@ -0,0 +1,56 @@
+\section{REPL commands}
+\label{sec:commands}
+
+To separate the implementation of user operations into logical units, each user
+operation is encapsulated within a class implementing the ``IReplCommand''
+interface. See \cref{fig:uml-commands} for the design of the commands package.
+After aquiring an invoker instance, the client has 2 commands available to him
+by default:
+
+\begin{description}
+  \item [:load] LanguageCommand, loads a language from a file path.
+  \item [:help] HelpCommand, prints descriptions of all available commands.
+\end{description}
+
+\begin{figure}[h]
+  \centering
+  \includegraphics[width=0.8\textwidth]{uml-commands}
```

Comment:
I think this can be a little bigger, I have to squint my eyes to see it. A good measure is to compare the text in the image with the text in the report. I think a little smaller is okay, but this is a bit too small in my opinion.